packages/blade/package.json

```diff
@@ -37,11 +37,13 @@
     "build": "run-s build:*",
     "build:clean": "rm -rf build",
     "build:generate-types": "tsc -p ./tsconfig-generate-types.json",
+    "build:generate-css-tokens": "FRAMEWORK=REACT rollup -c ./rollup-themes.config.js && node ./scripts/generateCSSTokens.js",
     "build:copy-declarations": "copyfiles -u 1 \"src/**/*.d.ts\" build/types",
     "build:react": "FRAMEWORK=REACT rollup -c && node ./scripts/generateRootImports.js",
     "build:react-native": "FRAMEWORK=REACT_NATIVE rollup -c && node ./scripts/generateRootImports.js",
     "build:declarations": "rollup -c && node ./scripts/generateRootImports.js",
     "build:clean-declarations": "rm -rf build/types",
+    "build:clean-theme-bundle": "rm -rf generated",
```

Comment:
why not use `build/css-vars`?

Reply:
This only cleans the themeBundle.js that is created by rollup. We dont need to publish it so it can't be in build folder

Reply:
i do that for types as well. I've build/types temporary and then clean it.